src/views/v2/widgets/events-list/event/cost.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Widget: Events List Event Cost
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/widgets/events-list/event/cost.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://m.tri.be/1aiy
+ *
+ * @version TBD
+ *
+ * @var WP_Post            $event   The event post object with properties added by the `tribe_get_event` function.
+ * @var array<string,bool> $display Associative array of display settings for event meta.
+ *
+ * @see tribe_get_event() For the format of the event object.
+ */
+
+if ( empty( $event->cost ) || empty( $display['cost'] ) ) {
```

Comment:
Just checking, we're not showing the cost here if the cost is `0`, `'0'` or any other value that would be marked as empty, is this what we want here?

Reply:
Good question, this is how it is in list view currently. I'll check.

Reply:
So for costs, if we enter `0` in the admin, it shows up as `Free` in the FE. If the field is left empty, it does not show.